webpack.config.js

```diff
@@ -1,30 +1,46 @@
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-const path = require('path');
-
-const SRC_DIR = path.join(__dirname, '/client');
-const DIST_DIR = path.join(__dirname, '/public');
+var path = require('path');
+var SRC_DIR = path.join(__dirname, '/client/src');
+var DIST_DIR = path.join(__dirname, '/client/dist');
 
 module.exports = {
   entry: `${SRC_DIR}/index.jsx`,
-  plugins: [
-    new HtmlWebpackPlugin({
```

Comment:
As per the last PR, why are we removing the HTML plugin? It generated the HTML file with all the required files for us.